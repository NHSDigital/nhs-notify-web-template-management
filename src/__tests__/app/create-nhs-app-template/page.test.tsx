/**
 * @jest-environment node
 */
import CreateNHSAppTemplatePage from '@app/create-nhs-app-template/page';

jest.mock('@forms/NhsAppTemplateForm/NhsAppTemplateForm');

describe('CreateNHSAppTemplatePage', () => {
  beforeEach(jest.resetAllMocks);

  it('should render CreateNHSAppTemplatePage', async () => {
    const page = await CreateNHSAppTemplatePage();

    expect(page).toMatchSnapshot();
  });
});
