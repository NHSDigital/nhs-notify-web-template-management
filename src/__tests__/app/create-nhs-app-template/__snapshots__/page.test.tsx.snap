// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateNhsAppTemplatePage page loads 1`] = `
<CreateNhsAppTemplate
  initialState={
    {
      "id": "session-id",
      "nhsAppTemplateMessage": "",
      "nhsAppTemplateName": "",
      "templateType": "NHS_APP",
    }
  }
/>
`;
